lib/perl/OVH/Bastion.pm

```diff
@@ -371,7 +371,7 @@ sub osh_header {
     my $hostname    = Sys::Hostname::hostname();
     my $versionline = 'the-bastion-' . $VERSION;
     my $output      = '';
-    if (OVH::Bastion::can_use_utf8()) {
+    if (OVH::Bastion::can_use_utf8() && grep { OVH::Bastion::config('fanciness')->value eq $_ } qw{ basic full }) {
```

Comment:
Suggestion : use this grep only once and cast it into a constant at the beginning of the file, to avoid code repetition

Reply:
I'd like to avoid instantiating a var at the beggining of the file without much context, however if/when there are more uses of the `fanciness` param at some point, I'll probably create a func "is_fanciness_equal_or_higher_than()", because that's almost always what I need to know